messages/extend.js

```diff
@@ -0,0 +1,89 @@
+const db = require('../db/connection');
+const { getSandboxNameFromMessage } = require('../common');
+
+function getSandboxDetails(channel, sandboxName) {
```

Comment:
A surprise we don't have it yet in `../common` 😅 But since we need it only here, I think we can keep it here 🤷‍♂️

Reply:
Yeah, yeah, agreed!